### ISSUE TITLE
Support 2 layers of ghost zones for rectilinear grids

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1768]
Rich Cook requested this on behalf of Andy Cook.

The issue is normals/lighting shows discontinuities at processor boundaries. 

Waiting on Rich for info on exactly how to reproduce this. 

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 04/10/2012 03:34 pm
Original update: 05/08/2012 02:39 pm
Ticket number: 1018
